javatests/jsinterop/generator/externs/overloads/overloads.js

```diff
@@ -0,0 +1,41 @@
+
+/**
+ * @fileoverview Test "overloaded" method conventions
+ * @externs
+ */
+
+/**
+ * @constructor
+ */
+function Foo() {}
+
+Foo.noArgMethod = function() {};
+Foo.prototype.noArgMethod = function() {};
```

Comment:
You can remove these two line, we already have these tests in simpleclass package

Reply:
Sounds good - my thinking was to have them all be covered here as well, so that one didn't have to scan all test packages to find other overload tests, but your point makes good sense too.